flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackendBuilder.java

```diff
@@ -397,7 +406,8 @@ private static void checkAndCreateDirectory(File directory) throws IOException {
 				nativeMetricOptions,
 				metricGroup,
 				restoreStateHandles,
-				ttlCompactFiltersManager);
+				ttlCompactFiltersManager,
+				writeBatchSize);
```

Comment:
Can we use `this.writeBatchSize` directly?

Reply:
I think this is equal to `this.writeBatchSize`, here we'll pass the `writeBatchSize` to another class `RocksDBIncrementalRestoreOperation`.